src/map/clif.c

```diff
@@ -8924,31 +8924,54 @@ void clif_slide(struct block_list *bl, int x, int y)
 /// 008d <packet len>.W <id>.L <message>.?B
 void clif_disp_overhead(struct block_list *bl, const char *mes)
 {
-	unsigned char buf[256]; //This should be more than sufficient, the theoretical max is CHAT_SIZE + 8 (pads and extra inserted crap)
+	struct PACKET_ZC_NOTIFY_CHAT *p;
+	uint32 max_len = CHAT_SIZE_MAX - sizeof(*p);
```

Comment:
Can you change this to `int` instead? (so that it's the same type as `int mes_len`)

`int max_len = CHAT_SIZE_MAX - (int)sizeof(*p);`